addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -459,7 +460,7 @@ export class Rtc extends Record {
      * @param {boolean} [initialState.camera]
      */
     async toggleCall(channel, { audio = true, camera } = {}) {
-        await loadJS("/mail/static/lib/selfie_segmentation/selfie_segmentation.js");
+        await loadJS(url("/mail/static/lib/selfie_segmentation/selfie_segmentation.js"));
```

Comment:
LGTM, HOOT is unhappy with the mocked URL failure...
Can add `.catch(() => {})` but then there's another HOOT failure of rpc performed after test and attempting a genuine RPC for `call_leave`:
<img width="1026" alt="Screenshot 2025-01-15 at 18 02 20" src="https://github.com/user-attachments/assets/df89fe24-5fdd-4033-a6e4-584dabfaf431" />

Wrapping a `Promise.resolve()` should do the trick:
```js
await Promise.resolve(() =>
            loadJS(url("/mail/static/lib/selfie_segmentation/selfie_segmentation.js")).catch(
                () => {}
            )
        );
```